src/Microsoft.Sbom.Api/Config/Args/GenerationArgs.cs

```diff
@@ -116,6 +116,15 @@ public class GenerationArgs : GenerationAndValidationCommonArgs
     [ArgDescription("If set to true, we will attempt to fetch license information of packages detected in the SBOM from the ClearlyDefinedApi.")]
     public bool? FetchLicenseInformation { get; set; }
 
+    /// <summary>
+    /// Specifies the timeout in seconds for fetching the license information. Defaults to 30 seconds. Has no effect if
+    /// FetchLicenseInformation (li) argument is false or not provided.
+    /// </summary>
+    [ArgShortcut("lto")]
+    [ArgDescription("Specifies the timeout in seconds for fetching the license information. Defaults to 30 seconds. " +
```

Comment:
Are any checks needed here?
- min value?
- max value?
- what if it's set when `FetchLicenseInformation` is false?

The parameter should be documented in both `docs\sbom-tool-arguments.md` and `src\Microsoft.Sbom.Targets\README.md`

Reply:
Thanks for the catch. The valid range is from `int.MinValue` to `int.MaxValue`. Negative values will be treated as InfiniteTimespan in the `LicenseInformationService`. When -lto is specified and `FetchLicenseInformation` is false, it simply has no effect. However, I did add a warning message for this case stating this.

I added test methods for sanitization of Min, Max, and null cases. I also tried specifying a value higher than `int.MaxValue` via command line argument and the program specified a warning message that the value exceeds the integer range, so we are good there as well (not sure how to automate that test).

Reply:
An infinite timeout seems unreasonably large. If actually applied, it would be a timeout of over 1600 years, which is probably not what you want in a build pipeline. Let's go with a smaller range--a max of 86400 (24 hours) seems more than adequate. If our supported range is 1-86400, then any other value should probably be a warning that we're ignoring their value and using the default, or an error that their value is out of range.